draft-vkrasnov-h2-compression-dictionaries.md

```diff
@@ -103,10 +103,73 @@ document are to be interpreted as described in RFC 2119 {{RFC2119}}.
 
 ## Security Considerations
 
-The use of compression over an encrypted connection is known to leak
-potentialy sensitive information. We will collaborate with industry experts to
-identify any additional attack vectors introduced by this draft, and include
-a set of best practices to both servers and clients that would implement it.
+The use of compression over an encrypted connection could be used by malicious
+actors to potentially leak sensitive information. We will collaborate with
+industry experts to identify any additional attack vectors introduced by this
+draft, and include a set of best practices to both servers and clients that
+would implement it.
+
+### Attack scenarios and mitigations
```

Comment:
Not entire sure if it belongs at the beginning, or the end of the document.

Reply:
I'm fine with either

Reply:
Let's keep it where it is for now